src/engine/engine.rs

```diff
@@ -18,11 +20,38 @@ use crate::{
         BlockDevTier, CreateAction, DeleteAction, DevUuid, FilesystemUuid, MaybeDbusPath, Name,
         PoolUuid, RenameAction, SetCreateAction, SetDeleteAction,
     },
-    stratis::StratisResult,
+    stratis::{ErrorEnum, StratisError, StratisResult},
 };
 
 pub const DEV_PATH: &str = "/stratis";
 
+/// The type of report for which to query.
```

Comment:
Can you move ```ReportType``` into ```types.rs```  (and update the header comment)?

Reply:
Resolved.